content/applications/inventory_and_mrp/inventory/barcode/operations/adjustments.rst

```diff
@@ -1,48 +1,141 @@
+:guilabel:`Default Nomenclature` or :guilabel:`Default GS1 Nomenclature` can be selected. The
+nomenclature selected changes how scanners interpret barcodes in Odoo.
 
-.. image:: adjustments/adjustments_04.png
-    :align: center
+There is also a :guilabel:`Configure Product Barcodes` internal link arrow, and a set of
+:guilabel:`Print` buttons for printing barcode commands and a barcode demo sheet.
 
-.. image:: adjustments/adjustments_05.png
-    :align: center
+.. image:: adjustments/adjustments-barcode-setting.png
+   :align: center
+   :alt: Enabled Barcode feature in Inventory app settings.
 
-When you have scanned all the items of the location, validate the
-inventory manually or scan the *Validate* barcode.
+For more on setting up and configuring the :guilabel:`Barcode` app, refer to the :doc:`Set up your
+barcode scanner </applications/inventory_and_mrp/inventory/barcode/setup/hardware>` and
+:doc:`Activate the Barcodes in Odoo
+</applications/inventory_and_mrp/inventory/barcode/setup/software>` documentation pages.
 
-.. image:: adjustments/adjustments_06.png
-    :align: center
+Create and apply inventory adjustment
+=====================================
+
+To scan product barcodes and apply inventory adjustments, navigate to the :menuselection:`Barcode
+app`.
+
+Once inside the :guilabel:`Barcode app`, a :guilabel:`Barcode Scanning` screen displaying different
+options is presented. To create and apply inventory adjustments, click on the :guilabel:`Inventory
+Adjustments` button at the bottom of the screen. This automatically creates a new adjustment, and
+navigates to a new Barcode Inventory Client Action page with :guilabel:`Inventory Adjustment` at the
+top of the form.
+
+.. image:: adjustments/adjustments-barcode-scanner.png
+   :align: center
+   :alt: Barcode app start screen with scanner.
+
+To begin the adjustment, first scan the source location in the warehouse of the product whose count
+is being adjusted. Then, scan the product barcode(s). The barcode of a specific product can be
+scanned multiple times to increase the quantity of that product in the adjustment.
+
+.. tip::
+   If the warehouse *multi-location* feature is not enabled in the database, a source location does
+   not need to be scanned. Instead, simply scan the product barcode to start the inventory
+   adjustment.
+
+Alternatively, the quantity can be changed by clicking the :guilabel:`pencil` icon on the far right
+of the product line, and editing the number in the :guilabel:`Quantity` line on the following
+screen. Additionally, the :guilabel:`+1` and :guilabel:`-1` buttons can be clicked to add or
+subtract quantity of the product, and the :guilabel:`number keys` can be used to add quantity, as
+well.
+
+.. example::
+   In the below Inventory Adjustment, the source location `WH/Stock/Shelf/2` was scanned. Then, the
+   barcode for the product `[FURN_7888] Desk Stand with Screen` was scanned 3 times. Additional
```

Comment:
Scanning the product does what? (: I think we explain just *a little* bit more about maybe how scanning the product increases the Quantity by 1, or something (: